src/pages/_document.tsx

```diff
@@ -6,7 +6,7 @@ export default function Document(props: DocumentProps) {
 
   return <Html lang={currentLocale}>
     <Head>
-      <meta charSet="utf-8" />
```

Comment:
This shouldn’t be removed

Reply:
NextJS seems to automatically add the tag anyway, including it here makes it that it's declared twice?